src/main/java/org/opentripplanner/street/model/edge/EscalatorEdge.java

```diff
@@ -0,0 +1,42 @@
+package org.opentripplanner.street.model.edge;
+
+import org.opentripplanner.framework.i18n.I18NString;
+import org.opentripplanner.street.model.vertex.Vertex;
+import org.opentripplanner.street.search.TraverseMode;
+import org.opentripplanner.street.search.state.State;
+
+/** Represents an escalator. An escalator edge can only be traversed by walking */
+public class EscalatorEdge extends Edge {
+
+  private double length;
+
+  private double speed = 0.45;
```

Comment:
Change this to be something like `private static final double HORIZONTAL_SPEED` and move this to be at the beginning of the class. Also, add javadoc for the field which explains how you have calculated the value.